aws-predictions/src/androidTest/java/com/amplifyframework/predictions/aws/TestConfiguration.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.predictions.aws;
+
+import android.content.Context;
+import androidx.annotation.NonNull;
+import androidx.test.core.app.ApplicationProvider;
+
+import com.amplifyframework.AmplifyException;
+import com.amplifyframework.core.Amplify;
+import com.amplifyframework.core.AmplifyConfiguration;
+import com.amplifyframework.testutils.Resources;
+import com.amplifyframework.testutils.sync.SynchronousMobileClient;
+
+import com.amazonaws.mobile.config.AWSConfiguration;
+
+/**
+ * This is a class to help configure Amplify and prepare other test resources
+ * for Predictions integration tests.
+ */
+final class TestConfiguration {
```

Comment:
Can we move over to `TestPredictionsCategory` factory style?

Reply:
Yes! I think your refactored logic is way cleaner and makes more sense.